dbms/tests/queries/0_stateless/01085_max_distributed_connections_http.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+query="SELECT sleepEachRow(1) FROM remote('127.{2,3}', system.one)"
+# 1.8 less then 2 seconds, but long enough to cover possible load peaks
+timeout 1.8s ${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&max_distributed_connections=2&max_threads=1&experimental_use_processors=1" -d "$query"
+timeout 1.8s ${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&max_distributed_connections=2&max_threads=1&experimental_use_processors=0" -d "$query"
```

Comment:
not sure if `experimental_use_processors` is required or "streams" build will add `experimental_use_processors=0` by itself.

Reply:
It will be added by default in separate check (release, streams).

Reply:
Ok, will be removed (wasn't sure that this is done for `CLICKHOUSE_URL`)